src/core/interp.c

```diff
@@ -1053,11 +1053,11 @@ void MVM_interp_run(MVMThreadContext *tc, void (*initial_invoke)(MVMThreadContex
             OP(param_op_o):
             {
                 MVMuint16 arg_idx = GET_UI16(cur_op, 2);
-                MVMArgInfo param = MVM_args_get_optional_pos_obj(tc, &tc->cur_frame->params, arg_idx);
-                if (param.exists) {
-                    GET_REG(cur_op, 0).o = param.arg.o;
+                MVMObject *param = MVM_args_get_optional_pos_obj(tc, &tc->cur_frame->params, arg_idx);
+                if (param) {
```

Comment:
I'm not sure if object registers may be NULLed. Could be the difference between "a parameter was passed" and "a defined parameter was passed". But I think in the undefined case, we'd still pass a type object (!IS_CONCRETE), so for objects this may actually fly.